sql/core/src/test/resources/sql-tests/inputs/variant-field-extractions.sql

```diff
@@ -0,0 +1,13 @@
+-- Simple field extraction and type casting.
+select parse_json('{ "price": 5 }'):price;
```

Comment:
nit: we can create a temp view with one or more VARIANT columns, to simplify the other SELECT queries in this test.